messaging/src/main/java/org/axonframework/eventhandling/pooled/Coordinator.java

```diff
@@ -131,8 +136,9 @@ private Coordinator(Builder builder) {
     public void start() {
         RunState newState = this.runState.updateAndGet(RunState::attemptStart);
         if (newState.wasStarted()) {
-            logger.debug("Starting Coordinator for Processor [{}].", name);
+            logger.debug("Optionally initializes token store and starting Coordinator for Processor [{}].", name);
```

Comment:
I don't we need this log change here, albeit true. If initialization does happen, the `private initializeTokenStore()` method will tell us so.

Reply:
reverted